### PR TITLE
Adding ability to pass in a KubeConfig

### PR DESCRIPTION
**Pull request by maxirus**
Adding the Ability to pass in a specific KubeConfig instead of relying on in cluster.

Also making info the default log_level as error didn't make sense.

### CODE REVIEW

File: sentry-kubernetes.py [hunk 1]
```diff
@@ -16,12 +16,12 @@
 # mapping from k8s event types to event levels
 LEVEL_MAPPING = {"normal": "info"}
 
-LOG_LEVEL = os.environ.get("LOG_LEVEL", "error")
+LOG_LEVEL = os.environ.get("LOG_LEVEL", "info")
```
Comment (HazAT) [inline sentry-kubernetes.py:19]: Why this change? I suppose this will produce a lot of noise.

**Review by HazAT — CHANGES_REQUESTED** [review]

File: sentry-kubernetes.py (hunk 1 above)
Comment (maxirus) [reply sentry-kubernetes.py:19]: LOG_LEVEL is being used to set the logging log level for this script [here](https://github.com/getsentry/sentry-kubernetes/blob/master/sentry-kubernetes.py#L54). Setting it to Warning makes the container report nothing to stdout when initializing.

**Comment by tonyo** [comment]
The agent has a completely different implementation now (written in Go), so I'll close this one as outdated.
